test/integration/emergency_broadcast_test.rb

```diff
@@ -3,6 +3,8 @@
 class EmergencyBroadcastTest < Capybara::Rails::TestCase
 
   setup do
+    # FIXME: This requires a VCR cassette that broken when I changed the super_admin's name in
+    # the fixture. web_mock should be used to block and mock all web requests.
```

Comment:
I'd like to propose an unofficial ban on VCR in our projects. It will become official once it's eliminated from all our projects.